### PR TITLE
Add the guides

### PR DESCRIPTION
**Pull request by bakulf**
In this PR I also had to add ordered-list and unordered-list types. And tests.

### CODE REVIEW

**Review by lesleyjanenorton — APPROVED** [review]
Looks great to me!

**Review by flodolo — COMMENTED** [review]
Left a few initial notes, but I need to spend a bit more time on this. 

Heads-up that here is public holiday until Tuesday, so this will have to wait for a bit.

File: src/ui/guides/01_how_to_vpn.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  "id": "how_to_vpn",
+  "title": "The top uses and benefits of using Mozilla VPN",
+  "image": "qrc:/guides/01_how_to_vpn.svg",
+  "blocks": [
+    { "id": "c_1",
+      "type": "text",
+      "content": "A VPN (or virtual private network) creates a secure and private connection by making a “tunnel” between your device and the web. Here are some of the top things you can do with Mozilla VPN." },
+    { "id": "c_2",
+      "type": "title",
+      "content": "Safeguard your personal data & identity" },
```
Comment (flodolo) [inline src/ui/guides/01_how_to_vpn.json:11]: https://mana.mozilla.org/wiki/display/FIREFOX/Word+List#WordList-ampersand/&

Should VPN follow the same guidelines?

File: src/ui/guides/02_is_my_vpn_working_correctly.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  "blocks": [
+    { "id": "c_1",
+      "type": "text",
+      "content": "After signing on to Mozilla VPN and connecting to one of our 400+ servers in over 30 different countries, you can check if your VPN is active in your chosen geolocation in two easy ways:" },
+    { "id": "c_2",
+      "type": "title",
+      "content": "Search Google" },
+    { "id": "c_3",
+      "type": "text",
+      "content": "Head to Google’s homepage and you will immediately see your location status (as well as any change in language) on display." },
```
Comment (flodolo) [inline src/ui/guides/02_is_my_vpn_working_correctly.json:14]: Not a localization issue, but this is off. 

Language on Google is influenced by several factors, location being only one of them. If you're logged in with your account, the language won't change and will follow your account's settings.

File: src/ui/guides/01_how_to_vpn.json (hunk 1 above)
Comment (bakulf) [reply src/ui/guides/01_how_to_vpn.json:11]: Good point.

File: src/ui/guides/02_is_my_vpn_working_correctly.json (hunk 2 above)
Comment (bakulf) [reply src/ui/guides/02_is_my_vpn_working_correctly.json:14]: Plus I don't like pushing for google in the VPN product. I'll discuss this with the UX team.

**Comment by flodolo** [comment]
For review it would be great to have separate follow-up commits, and squash them before merge.

For example, is there a reason for [this change](https://github.com/mozilla-mobile/mozilla-vpn-client/compare/acb38fcdc954f095bd81cbbd7b0efa0b8904a63b..7ced69b091023a2a0c7beb6c1f86d0ad1095f445#diff-b293d0715122f412eb5fa3c4ee7470c4c5a4baa6b42569a0f6df8bb2586f2201R26), or going back to "Personal" uppercase?

**Review by lesleyjanenorton — CHANGES_REQUESTED** [review]
It looks like some of the strings have changed since the version of the spec these strings were pulled from. Final copy is here https://www.figma.com/file/LmWggJlWXCi23jOpitIDnA/VPN-Tips-%26-Tricks?node-id=316%3A15328 under "GTM release v2.9"

**Comment by bakulf** [comment]
New strings

File: src/ui/guides/01_how_to_vpn.json [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+      ]
+    },
+    { "id": "c_3",
+      "type": "title",
+      "content": "Defense on the go" },
+    { "id": "c_4",
+      "type": "ulist",
+      "content": [
+        { "id": "l_1",
+          "content": "Blocks unknown entities from seeing your private data on risky public wifi and hotspots" },
```
Comment (flodolo) [inline src/ui/guides/01_how_to_vpn.json:26]:
```suggestion
          "content": "Blocks unknown entities from seeing your private data on risky public Wi-Fi and hotspots" },
```

**Review by lesleyjanenorton — APPROVED** [review]

File: src/ui/guides/04_connecting_external_devices.json [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    { "id": "c_1",
+      "type": "title",
+      "content": "How do I connect to my printer or fax?" },
+    { "id": "c_2",
+      "type": "olist",
+      "content": [
+        { "id": "l_1",
+          "content": "Enable “Local network access” in your network settings" },
+        { "id": "l_2",
+          "content": "Connect the printer or fax cable to your device’s port (wired) OR via wireless" },
```
Comment (flodolo) [inline src/ui/guides/04_connecting_external_devices.json:16]: Out of curiosity, why `OR`? All the other are `or`

File: src/ui/guides/04_connecting_external_devices.json [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+        { "id": "l_3",
+          "content": "Print or fax your documents as normalDesktop computers / laptops / mobile phones" }
```
Comment (flodolo) [inline src/ui/guides/04_connecting_external_devices.json:18]:
```suggestion
          "content": "Print or fax your documents as normal Desktop computers / laptops / mobile phones" }
```

Missing a space

File: src/ui/guides/04_connecting_external_devices.json [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+    { "id": "c_4",
+      "type": "text",
+      "content": "With “Local network access” turned on, you can also connect to devices like:" },
+    { "id": "c_5",
+      "type": "ulist",
+      "content": [
+        { "id": "l_1",
+          "content": "Other computers and mobile phones" },
+        { "id": "l_2",
+          "content": "Streaming sticks (Chromecast / Roku / Apple Airplay)" }
```
Comment (flodolo) [inline src/ui/guides/04_connecting_external_devices.json:33]:
```suggestion
          "content": "Streaming sticks (Chromecast / Roku / Apple AirPlay)" }
```

nit (see [Apple's page](https://www.apple.com/airplay/))

**Review by flodolo — APPROVED** [review]
A few nits to fix, but looks good to me.

**Pull request merged**